STYLE.md

```diff
@@ -127,12 +127,36 @@ spelled the same.
 
 ## Units
 
-- **Currency:** all amounts should be denominated in BTC.  E.g. "the
-  default minimum relay fee is 0.00000001 BTC/vbyte".  When writing out
-  the name of the currency, pluralize by appending an `s`.  E.g. "Alice
-  owns several bitcoins" not ~~"Alice owns several bitcoin"~~.
-
-- **Weight:** use vbytes rather than weight units, unless using weight
+- **Currency:** acceptable units and abbreviations are:
+
+    | Unit | Abbreviations | Value |
+    |-|-|-|
+    | bitcoins | **BTC** | |
+    | millibitcoins | mBTC | 0.001 BTC |
+    | microbitcoins | µBTC, uBTC | 0.000001 BTC |
+    | satoshis | **sat** | 0.00000001 BTC |
+    | nanobitcoins | nBTC | 0.000000001 BTC |
+    | millisatoshis | msat | 0.00000000001 BTC |
+    | picobitcoins | pBTC | 0.000000000001 BTC |
```

Comment:
perhaps add a fourth column in exponential/scientfic notation (I find it helpful for expressing and recalling these units and the conversions between them)